octavia_f5/controller/worker/sync_manager.py

```diff
@@ -162,8 +164,6 @@ def tenant_update(self, network_id, device=None):
 
         loadbalancers = self._loadbalancer_repo.get_all_by_network(
             db_apis.get_session(), network_id=network_id, show_deleted=False)
-        if not loadbalancers:
-            return False
```

Comment:
Please don't remove that, it's a safeguard prohibiting to update a tenant without any elements (if that action was requested, tenant_delete() intended). This could for example happen by uncorrect usage of manual `octavia-f5-util` sync.